cli/src/main.rs

```diff
@@ -62,6 +62,14 @@ fn run() -> Result<(), CliError> {
                 (@arg quiet: -q --quiet "do not display output")
             )
         )
+        (@subcommand health =>
+            (about: "displays information about network health")
```

Comment:
The first letter should be capitalized so output in "carto run -- --help" is consistent.

Reply:
This description probably becomes outdated when we add the active tests. Maybe "Health commands" to match "Administrative commands", though something slightly more descriptive (but not verbose) might be better.

Reply:
If that's the case, it should be done in an additional commit, as the other messages are lowercase